### ISSUE TITLE
ContributionGroup.qml has incorrect permissions set

### ISSUE THREAD

**Issue by llewelld** [Pretty-SFOS/opal-about#2]
In the latest 1.0.0 release archive the default permissions for `ContributionGroup.qml` are set to `600`. The default Sailfish SDK doesn't set the permissions in the spec file, so if the archive is unpacked directly into a project, these permissions will be transferred over to the app install.

As a result, running the app results in the following error, which prevents the About page from opening:
```bash
[W] unknown:1283 - file:///usr/lib64/qt5/qml/Sailfish/Silica/PageStack.qml:1283:13: QML AnimatedLoader: (file:///usr/share/harbour-opaltest/qml/pages/About.qml:5:1: Type A.AboutPageBase unavailable
    A.AboutPageBase {
    ^, file:///usr/share/harbour-opaltest/qml/modules/Opal/About/AboutPageBase.qml:9:1: Type ContributionSection unavailable
    Page{id:page
    ^, file:///usr/share/harbour-opaltest/qml/modules/Opal/About/ContributionSection.qml:6:1: Type ContributionGroup unavailable
    QtObject{property string title
    ^, file:///usr/share/harbour-opaltest/qml/modules/Opal/About/ContributionGroup.qml: File not found)
```
All of the other files are set to `622`, which works fine.

Here's the result from a clean unpacking of the folder for reference:
```bash
flypig@chattan:~/temp$ wget https://github.com/Pretty-SFOS/opal-about/releases/download/v1.0.0/opal-about-1.0.0-e55253d.tar.gz
flypig@chattan:~/temp$ sha256sum opal-about-1.0.0-e55253d.tar.gz
c1e2e3c797c63489bc8e9ce93af0fbd480a6f31624f814d851e4bef5c5520d3c  opal-about-1.0.0-e55253d.tar.gz
flypig@chattan:~/temp$ tar -xvf opal-about-1.0.0-e55253d.tar.gz 
flypig@chattan:~/temp$ ls -l opal-about/qml/modules/Opal/About/
total 56
-rw-r--r-- 1 flypig flypig 6251 Jun  3 19:08 AboutPageBase.qml
-rw-r--r-- 1 flypig flypig  760 Jun  3 19:08 Attribution.qml
-rw-r--r-- 1 flypig flypig  360 Jun  3 19:08 ChangelogList.qml
-rw-r--r-- 1 flypig flypig  577 Jun  3 19:08 Changelog.qml
-rw------- 1 flypig flypig  355 Jun  3 19:08 ContributionGroup.qml
-rw-r--r-- 1 flypig flypig  273 Jun  3 19:08 ContributionSection.qml
-rw-r--r-- 1 flypig flypig  248 Jun  3 19:08 DonationService.qml
-rw-r--r-- 1 flypig flypig  273 Jun  3 19:08 InfoButton.qml
-rw-r--r-- 1 flypig flypig 3445 Jun  3 19:08 InfoSection.qml
-rw-r--r-- 1 flypig flypig 1481 Jun  3 19:08 License.qml
-rw-r--r-- 1 flypig flypig  402 Jun  3 19:08 OpalAboutAttribution.qml
drwxr-xr-x 2 flypig flypig 4096 Jun  3 19:08 private
-rw-r--r-- 1 flypig flypig  568 Jun  3 19:08 qmldir
```

**Comment by ichthyosaurus** [comment]
Thank you! I fixed this in the release script ([commit](https://github.com/Pretty-SFOS/opal/commit/01e3a10fb176890f1bb04307d618743e2fa767da)), released a new version ([v2.0.0](https://github.com/Pretty-SFOS/opal-about/releases/tag/v2.0.0)), and uploaded a fixed bundle for [v1.0.0](https://github.com/Pretty-SFOS/opal-about/releases/tag/v1.0.0).

**Comment by llewelld** [comment]
Great! Thanks for your quick work!
